lib/diagnostic.js

```diff
@@ -39,20 +50,33 @@ const owner = null
 // Methods: vocalize
 
 // Replace null with your answer
-const pet = null
+const pet = {
+  name: 'Max',
+  species: 'cat',
+  breed: 'Maine Coon',
+  vocalization: 'Yoooowwwl!',
+  vocalize: function () {
+    return (this.vocalization)
+  }
+}
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function () {
-  /* Your answer here */
```

Comment:
Here we were looking for something that would take 4 parameters, and then assign those to the object that is returned. Something like this: 
`const Pet = function (name, species, breed, vocalization) {
  this.name = name
  this.species = species
  this.breed = breed
  this.vocalization = vocalization
}`